README.rst

```diff
@@ -166,13 +166,15 @@ Requirements
 - charset-normalizer_
 - multidict_
 - yarl_
+- fozenlist_
 
 Optionally you may install the cChardet_ and aiodns_ libraries (highly
 recommended for sake of speed).
 
-.. _charset-normalizer: https://pypi.org/project/charset-normalizer
+.. _charset-normalizer: https://pypi.python.org/project/charset-normalizer
```

Comment:
This change is incorrect.

Reply:
Why is it incorrect? It redirects to the regular URL anyway

Reply:
The canonical domain is pypi.org.